src/Athens-Examples/AthensCairoSurfaceExamples.class.st

```diff
@@ -76,8 +76,8 @@ A cairo image surface bits are exposed to bitblt operations via surface plugin.
 		can setShape: (-20@ -20 corner: 20@ 20).
 		2 timesRepeat: [ can draw. can setPaint: (Color green alpha:0.5)]
 	].
-	Display getCanvas drawImage: 	surf asForm  at: 0@0
 
+	surf asForm asMorph openInWindow
```

Comment:
Could it be

`surf asMorph openInWindow`?

Or do we want that in general surfaces can be transformed into forms?